threadit-api/Controllers/v1/SpoolController.cs

```diff
@@ -59,11 +59,76 @@ public async Task<IActionResult> GetAllSpools([FromServices] SpoolService spoolS
             return Ok(spools);
         }
 
+        [HttpGet("delete/{spoolId}")]
+        public async Task<IActionResult> DeleteSpool([FromRoute] string spoolId, [FromServices] SpoolService spoolService)
```

Comment:
- add `[AuthenticationRequired]` attribute
- check and make sure calling user ID (`request.GetUser()`) matches the current owner before changing